app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingViewModelTest.kt

```diff
@@ -942,6 +942,26 @@ class VaultItemListingViewModelTest : BaseViewModelTest() {
             }
         }
 
+    @Suppress("MaxLineLength")
+    @Test
+    fun `AddVaultItemClick inside a folder should emit NavigateToAddVaultItem with a selected folder id`() = runTest {
```

Comment:
Formatter, there are instructions on how to setup the macro for this [here](https://github.com/bitwarden/android?tab=readme-ov-file#setup)

Reply:
Sorry I had it configured already I just forgot about it thinking that the `gradle detekt` would apply our code styling as well.